roles/sales-rep/accounts-view/BezlExport.txt

```diff
@@ -600,15 +605,27 @@
     },
     {
       "method": "setConfig",
-      "code": "bezl.vars.config = {};\n\nbezl.vars.config.baseUrl = \"\";\n// If the baseURL is not set in the line above then we will determine a sane default based on the URL\nif (bezl.vars.config.baseUrl == \"\") {\n    if (window.location.href.indexOf(\"portal-dev.bezl.io\") !== -1) {\n        bezl.vars.config.baseUrl = \"https://bezlio-apps-dev.bezl.io\";\n    } else {\n        bezl.vars.config.baseUrl = \"https://bezlio-apps.bezl.io\";\n    }\n}\nbezl.vars.config.baseJsUrl = bezl.vars.config.baseUrl + \"/roles/sales-rep/accounts-view/js/\"\nbezl.vars.config.baseLibraryUrl = bezl.vars.config.baseUrl + \"/libraries/\"",
+      "code": "bezl.vars.config = {};\n\nbezl.vars.config.baseUrl = \"\";\nif(bezl.vars.config.baseUrl == \"\") {\n  \tif(window.location.href.indexOf(\"portal-dev.bezl.io\") !== -1) {\n     \tbezl.vars.config.baseUrl = \"https://bezlio-apps-dev.bezl.io\";\n    } else {\n     \tbezl.vars.config.baseUrl = \"https://bezlio-apps.bezl.io\"; \n    }\n}\n\nbezl.vars.config.baseUrl = \"https://rawgit.com/bezlio/bezlio-apps/Sales-Rep-Accounts-View-Add-Category-Selectors\";\n\nbezl.vars.config.baseJsUrl = bezl.vars.config.baseUrl + \"/roles/sales-rep/accounts-view/js/\";\nbezl.vars.config.baseLibraryUrl = bezl.vars.config.baseUrl + \"/libraries/\";",
```

Comment:
@soft-Dev I'm sorry to create more work for you but can you take out the RawGit line in the setConfig() function?